charmcraft/extensions/gunicorn.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2023 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For further info, check https://github.com/canonical/charmcraft
+
+"""The flask extension."""
+import abc
+from typing import Any
+
+from overrides import override
+
+from ..errors import ExtensionError
+from .extension import Extension
+
+_ACTIONS = {
+    "flask": {
+        "rotate-secret-key": {
+            "description": "Rotate the flask secret key. Users will be forced to log in again. This might be useful if a security breach occurs."
+        }
+    },
+    "django": {
+        "rotate-secret-key": {
+            "description": "Rotate the flask secret key. Users will be forced to log in again. This might be useful if a security breach occurs."
```

Comment:
```suggestion
            "description": "Rotate the django secret key. Users will be forced to log in again. This might be useful if a security breach occurs."
```

I presume? (Or make it generic by removing the framework name)